app/src/main/java/me/tangobee/weathernaut/ui/Next7DaysFragment.kt

```diff
@@ -99,6 +99,18 @@ class Next7DaysFragment : Fragment() {
         }
 
         val days = arrayOf("Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday")
+        val days_es = arrayOf("Domingo", "Lunes", "Martes", "Miércoles", "Jueves", "Viernes", "Sábado")
+        val days_fr = arrayOf("Dimanche", "Lundi", "Mardi", "Mercredi", "Jeudi", "Vendredi", "Samedi")
+        val days_de = arrayOf("Sonntag", "Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag")
```

Comment:
Can you move them to `string.xml` just like the weather constants?

Reply:
the days variable on line 101 was already present and I just added spanish, french, german days. Still should we move it? because it is an array, so day retrieval is much easier. What do you say?

Reply:
Okay, let's leave it as is for now.